packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -527,30 +474,11 @@ class DevFS {
           assetPathsToEvict.add(archivePath);
       }
     });
-    // We run generator even if [dirtyEntries] was empty because we want to
-    // keep logic of accepting/rejecting generator's output simple: we must
-    // accept/reject generator's output after every [update] call.  Incremental
-    // run with no changes is supposed to be fast (considering that it is
-    // initiated by user key press).
-    final List<String> invalidatedFiles = <String>[];
-    final Set<Uri> filesUris = <Uri>{};
-    for (Uri uri in dirtyEntries.keys.toList()) {
-      if (!uri.path.startsWith(assetBuildDirPrefix)) {
-        final DevFSContent content = dirtyEntries[uri];
-        if (content is DevFSFileContent) {
-          filesUris.add(uri);
-          invalidatedFiles.add(content.file.uri.toString());
-          syncedBytes -= content.size;
```

Comment:
How syncedBytes is going to be updated with this code gone?

Reply:
I added syncedBytes which I accidentally removed from the dill sync. All other entires are assets and are handled above